meta: document overriding custom server jar (#98)
Infinidoge/nix-minecraft: README.md

- `velocity-server`: Same as `velocityServers.velocity`
- `minecraft-server`: Same as `vanilla-server`

> Server versions not found above can be manually via an override.

Reviewer: ```suggestion
Server versions not found above can be setup manually via an override.
```

> This override changes the path the launch script uses to your provided jar file, and does not modify the vanilla jar.
> In this example, uberbukkit.jar exists next to this configuration:

Reviewer: ```suggestion
For example, this override changes the path the launch script uses to your provided jar file, and does not modify the vanilla jar:
```

I don't think it's necessary to specify that the jar is alongside, since that should be implicit in how the code is written.
